tests/atest/rules/spacing/bad_indent/expected_output.txt

```diff
@@ -1,40 +1,40 @@
-bug375.robot:7:5 [W] 1008 Line is under-indented
-bug375.robot:8:5 [W] 1008 Line is under-indented
-bug375.robot:23:7 [W] 1008 Line is over-indented
-bug375.robot:33:4 [W] 1008 Line is under-indented
-comments.robot:19:3 [W] 1008 Line is over-indented
-comments.robot:23:4 [W] 1008 Line is under-indented
-comments.robot:26:5 [W] 1008 Line is over-indented
+bug375.robot:7:1 [W] 1008 Line is under-indented
```

Comment:
Now we're showing the beginning of the line instead of the end of bad indent?

Reply:
Yes, because the indentation starts at the beginning of the line.
Note, that these changes are for the LSP in IDE, so that the issues are properly underlined in the code editor.

<img width="504" alt="image" src="https://user-images.githubusercontent.com/7412964/226835812-bc85c3fa-88c2-458a-a56b-0bdb140e6eb6.png">